Server/JokesOnYou.Web.Api/Services/UserService.cs

```diff
@@ -37,5 +44,40 @@ public async Task UpdateUser(UserUpdateDTO updateDTO)
         {
             throw new NotImplementedException();
         }
+
+        public async Task<UserReplyDTO> LoginUser(UserLoginDTO userLogin)
+        {
+            var user = await _userRepository.GetUserByEmail(userLogin.Email);
+            if (user != null)
+            {
+                var signInResult = await _signInManager.CheckPasswordSignInAsync(user, userLogin.Password, false);
+
+                if (signInResult.Succeeded)
```

Comment:
I would recommend always doing i think its called inverted if statements. So instead of checking if it succeeded, check if it failed and just throw an exception. Will keep the code clean